eva-lib/src/main/java/uk/ac/ebi/eva/lib/filter/FilterBuilder.java

```diff
@@ -34,12 +34,17 @@
                                                                                  String polyphenScore,
                                                                                  String siftScore,
                                                                                  List<String> studies,
-                                                                                 List<String> consequenceType) {
+                                                                                 List<String> consequenceType,
+                                                                                 String ref, List<String> alts,
+                                                                                 List<String> xrefIds) {
```

Comment:
I don't think xrefs should be considered a filter, because queries are either by region or gene, but can't be by both.